ecommerce/static/templates/_course_credit_seats.html

```diff
@@ -12,7 +12,7 @@
             <thead>
                 <tr>
                     <th><%= gettext('Credit Provider') %></th>
-                    <th><%= gettext('Price (USD)') %></th>
+                    <th><%= gettext('Price ('+ currency +')') %></th>
```

Comment:
nit: you could have also used ${currency}

Reply:
I tried to use it, but it wasn't working.